language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/command/CommandUtil.java

```diff
@@ -238,12 +328,41 @@ public String getArgumentV() {
         }
     }
 
-    private static String getDocumentationAttribute(String docTag, String field) {
-        return String.format("%s{{%s}}", docTag, field);
+    private static String getDocumentationAttribute(String docTag, String field, int offset) {
+        String offsetStr = String.join("", Collections.nCopies(offset, " "));
+        return String.format("%s%s{{%s}}", offsetStr, docTag, field);
+    }
+
+    private static String getDocumentationAttachment(List<String> attributes, int offset) {
+        String offsetStr = String.join("", Collections.nCopies(offset, " "));
+        if (attributes == null || attributes.isEmpty()) {
+            return String.format("%n%sdocumentation {%n%s\t%n%s}", offsetStr, offsetStr, offsetStr);
+        } else {
```

Comment:
Else is not required as method returns within if scope

Reply:
Will fix the same with "Adding documentation for the whole document"